project-base/storefront/components/Blocks/Product/Filter/FilterPanel.tsx

```diff
@@ -24,80 +23,82 @@ const DEFAULT_NUMBER_OF_SHOWN_FLAGS = 5;
 const DEFAULT_NUMBER_OF_SHOWN_BRANDS = 5;
 const DEFAULT_NUMBER_OF_SHOWN_PARAMETERS = 5;
 
-export const FilterPanel = memo<FilterPanelProps>(
-    ({ productFilterOptions: filterOptions, panelCloseHandler, totalCount }) => {
-        const { t } = useTranslation();
+export const FilterPanel: FC<FilterPanelProps> = ({
```

Comment:
I am not saying it is wrong or that I do not agree, just wanted to know what was your motivation for removing the memo wrapper?

Reply:
Maybe I would ask what was motivation for use memo here?

Reply:
Yeah, good question 😆